cypress/support/commands.ts

```diff
@@ -16,15 +17,31 @@ export const signin = (): Cypress.Chainable<Cypress.Response> => {
     })
   \*/
 
-  return cy.setupUser().then(body => {
-    return cy
-      .visit('/api/v2/signin')
-      .then(() => cy.get('#login').type(Cypress.env('username')))
-      .then(() => cy.get('#password').type(Cypress.env('password')))
-      .then(() => cy.get('#submit-login').click())
-      .then(() => cy.get('.theme-btn--success').click())
-      .then(() => cy.wrap(body))
```

Comment:
Previously--- all tests depend on what is returned from this 'body' returned from setupUser which is the response to a 'GET' request to `/debug/provision`